scripts/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "internal-scripts",
+  "version": "0.0.0-development",
```

Comment:
This is an un-related drive-by change. There's a warning in the console any time we run any task saying that `scripts` is missing a version. This is just to make that warning go away

Reply:
Been wanting to do this as well